Classes/Classes/miniphonedir.cpp

```diff
@@ -0,0 +1,101 @@
+#include<iostream> 
+
+using namespace std; 
+
+  
+
+class UserData { 
+
+private: 
+
+    string name; 
```

Comment:
Better to always use std::string and remove using namespace std; to avoid name collision